packages/shared/src/relay/getPreviewImageUrlsInlineDangerously.ts

```diff
@@ -238,15 +242,7 @@ export function getPreviewImageUrlsInlineDangerously({
 
   if (!previewUrls) {
     return {
-      type: 'error',
-      error: new CouldNotRenderNftError(
-        'getPreviewImageUrlsInlineDangerously',
-        'No preview URLs returned for token!',
-        {
-          id: token?.dbid,
-          assetType: media?.__typename,
-        }
-      ),
+      type: 'temp',
```

Comment:
I don't have a concrete fix yet for this issue, need to investigate it more deeper. 

Less hacky stuff could be like this. We just ignore the error only for the specific user. 

We could filter it by `username` or user id` or use `UnknownMedia` as an indicator

Example

```tsx
  if (!previewUrls && token.owner?.username === 'welovetheart') {
    return {
      type: 'ignore-invalid-media',
    };
  }

```